apps/judicial-system/web/src/shared-components/AccordionItems/CourtRecordAccordionItem/CourtRecordAccordionItem.tsx

```diff
@@ -138,35 +138,53 @@ const CourtRecordAccordionItem: React.FC<Props> = ({ workingCase }: Props) => {
           <Text>{formatMessage(m.sections.conclusion.disclaimer)}</Text>
         </Box>
       )}
-      <Box component="section" marginBottom={3}>
-        <AccordionListItem title="Ákvörðun um kæru">
-          <Box marginBottom={2}>
-            <Text>{formatMessage(m.sections.appealDecision.disclaimer)}</Text>
-          </Box>
-          {workingCase.prosecutorAppealDecision !==
-            CaseAppealDecision.NOT_APPLICABLE && (
-            <Box marginBottom={1}>
+      {(isRestrictionCase(workingCase.type) ||
+        workingCase.sessionArrangements === SessionArrangements.ALL_PRESENT ||
```

Comment:
Wondering if we could move this heavy business to a descriptive helper function for better readability?